app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getCategoriesC } = require("./controllers/controllers");
+
+const app = express();
+app.use(express.json());
```

Comment:
I dont think if I removed this line of code it would break any of your tests, which suggests to me it doesnt need to be in here yet 🤔